src/factorial.js

```diff
@@ -0,0 +1,13 @@
+export default function factorial(num){
+	if(num > 0){
+		var array = []
+		while(num > 0){
+			array.push(num)
+			num--}
+		return array.reduce(function(a,b){return a*b})
+	}else if(num === 0){
+		return 1
+	}else{
+		return 'Invalid input'
+	}
+}
```

Comment:
This code also works well. Good job using the while loop, when I was working on this, I did something different. For sure, the while loop looks cleaner than what I did.